tslearn/tests/test_metrics.py

```diff
@@ -42,6 +43,24 @@ def test_dtw_subseq():
     np.testing.assert_equal(path, [(0, 2), (1, 3)])
     np.testing.assert_allclose(dist, 0.)
 
+    path, dist = tslearn.metrics.dtw_subsequence_path([1, 4],
+                                                      [1., 2., 2., 3., 4.])
+    np.testing.assert_equal(path, [(0, 2), (1, 3)])
+    np.testing.assert_allclose(dist, np.sqrt(2.))
+
+
+def test_dtw_subseq_path():
+    subseq, longseq = [1, 4], [1., 2., 2., 3., 4.]
+    subseq = to_time_series(subseq)
+    longseq = to_time_series(longseq)
+    cost_matrix = tslearn.metrics.njit_accumulated_matrix_subsequence(subseq, longseq)
+
+    path = tslearn.metrics.return_path_subsequence(cost_matrix, 3)
+    np.testing.assert_equal(path, [(0, 2), (1, 3)])
+
+    path = tslearn.metrics.return_path_subsequence(cost_matrix, 1)
+    np.testing.assert_equal(path, [(0, 0), (1, 1)])
+
```

Comment:
Maybe adding a test that asserts that on some randomly-generated data, `dtw_subsequence_path` gives the same path / cost as `return_path_subsequence` when starting from the argmin?